### PR TITLE
Webdriver framework python

### PR DESCRIPTION
**Pull request by princebaretto99**
Added python version of Webdriver framework.

### CODE REVIEW

File: webdriver-framework-core-python/src/main/webdriver/core/LocalFactory.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+class LocalFactoryClass:
+    instance = None 
+
+    local = Local()
+    localIdentifier : str = ""
+
+    LocalFactoryClassLock = threading.Lock()
+
+    #ENV variable to be set:
```
Comment (AnubhavBasu) [inline webdriver-framework-core-python/src/main/webdriver/core/LocalFactory.py]: Remove the commented lines.

Comment (princebaretto99) [reply webdriver-framework-core-python/src/main/webdriver/core/LocalFactory.py]: Done

**Review by AnubhavBasu — APPROVED** [review]

File: selenium/webdriver-framework-core-python/src/test/resources/webdriver-config.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+driverType: cloudDriver
+
+onPremDriver:
+  platforms:
+    - name:
+      browser: Chrome
+      driverPath: /Users/princeton99/Downloads/Drivers/ChromeDriver/chromedriver
+    - name:
+      browser: Firefox
+      driverPath: /Users/princeton99/Downloads/Drivers/Mozilla/geckodriver
```
Comment (rathilpatel) [inline selenium/webdriver-framework-core-python/src/test/resources/webdriver-config.yml]: remove personal directory, add sample directory, add comments for information

File: selenium/webdriver-framework-core-python/src/main/webdriver/core/WebDriverFactory.py [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+
+from selenium import webdriver
+
+class WebDriverFactoryClass:
+    DEFAULT_CAPABILITIES_FILE = "PATH_TO_YML_FILE"
+
+    BROWSERSTACK_USERNAME = "BROWSERSTACK_USERNAME"
+    BROWSERSTACK_ACCESS_KEY = "BROWSERSTACK_ACCESS_KEY"
+    BUILD_ID = "BUILD_ID"
+    DEFAULT_BUILD_NAME = "browserstack-examples-junit5"
```
Comment (rathilpatel) [inline selenium/webdriver-framework-core-python/src/main/webdriver/core/WebDriverFactory.py]: browserstack-example-robot

File: selenium/webdriver-framework-core-python/README.MD [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      os_version: Big Sur
+      browser: Chrome
+      browser_version: latest
+      capabilities:
+        browserstack.selenium_version: 3.141.59
+```
+
+## Initialize webdriver instances based on the configuration
+
+The [WebDriverFactory](src/main/webdriver/core/WebDriverFactory.java) class looks for environment variable named "capabilities.config" which should be pointing to the file to be used. Further the class parses the file and initialises the [WebDriverConfiguration](src/main/webdriver/config/WebDriverConfiguration.java) class
```
Comment (rathilpatel) [inline selenium/webdriver-framework-core-python/README.MD]: Make changes for Python

File: selenium/webdriver-framework-core-python/src/main/webdriver/core/WebDriverFactory.py (hunk 3 above)
Comment (princebaretto99) [reply selenium/webdriver-framework-core-python/src/main/webdriver/core/WebDriverFactory.py]: Done

File: selenium/webdriver-framework-core-python/src/test/resources/webdriver-config.yml (hunk 2 above)
Comment (princebaretto99) [reply selenium/webdriver-framework-core-python/src/test/resources/webdriver-config.yml]: Done

File: selenium/webdriver-framework-core-python/README.MD (hunk 4 above)
Comment (princebaretto99) [reply selenium/webdriver-framework-core-python/README.MD]: Done

File: selenium/webdriver-framework-core-python/README.MD [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+      driverPath: src/test/resources/safaridriver
+cloudDriver:
+  hubUrl: https://hub-cloud.browserstack.com/wd/hub
+  user: BROWSERSTACK_USERNAME
+  accessKey: BROWSERSTACK_ACCESSKEY
+  localTunnel:
+    enabled: false
+  common_capabilities:
+    project: BrowserStack Demo Repository
+    buildPrefix: browserstack-examples-testng
```
Comment (rathilpatel) [inline selenium/webdriver-framework-core-python/README.MD]: browserstack-example-robot

Comment (princebaretto99) [reply selenium/webdriver-framework-core-python/README.MD]: done

**Review by rathilpatel — APPROVED** [review]

**Pull request closed without merging**